packages/@storefront/navigation/src/filter-refinement-controls/index.ts

```diff
@@ -7,11 +8,24 @@ class FilterRefinementControls extends ValueRefinementControls {
     return 'filterControls';
   }
 
+  get selectMultipleRefinements(): string {
```

Comment:
I'm not sure this is a good name for this method. If I understand correctly, this getter computes the name of the action creator, rather than actually selecting anything?

Reply:
This was done in order to keep consistency with the getter methods inside of ValueRefinementControls, which this class is extended from.